cpp/src/barretenberg/stdlib/primitives/safe_uint/safe_uint.test.cpp

```diff
@@ -1,62 +1,66 @@
 
 #include <cstddef>
+#include <gtest/gtest-typed-test.h>
 #include <gtest/gtest.h>
+#include "barretenberg/stdlib/primitives/bool/bool.hpp"
+#include "barretenberg/stdlib/primitives/witness/witness.hpp"
 #include "safe_uint.hpp"
 #include "barretenberg/numeric/random/engine.hpp"
 #include "../byte_array/byte_array.hpp"
 #include "barretenberg/honk/composer/standard_honk_composer.hpp"
 #include "barretenberg/stdlib/types/types.hpp"
 
+#pragma GCC diagnostic ignored "-Wunused-local-typedefs"
+
+#define STDLIB_TYPE_ALIASES                                                                                            \
+    using Composer = TypeParam;                                                                                        \
+    using witness_ct = stdlib::witness_t<Composer>;                                                                    \
+    using field_ct = stdlib::field_t<Composer>;                                                                        \
+    using bool_ct = stdlib::bool_t<Composer>;                                                                          \
+    using suint_ct = stdlib::safe_uint_t<Composer>;                                                                    \
+    using byte_array_ct = stdlib::byte_array<Composer>;                                                                \
+    using public_witness_ct = stdlib::public_witness_t<Composer>;
+
 namespace {
 auto& engine = numeric::random::get_debug_engine();
 }
 
-using namespace proof_system::plonk::stdlib::types;
-
 namespace test_stdlib_safe_uint {
+using namespace barretenberg;
+using namespace plonk;
```

Comment:
When you rebase this will probably cause you problems, since there is no top-level namespace `plonk` since yesterday, so instead you should use namespace `proof_system::plonk`.

Reply:
this thankfully rebased correctly by itself